doc/source/google/step-zero-gcp.rst

```diff
@@ -84,7 +84,11 @@ your google cloud account.
    * ``--zone`` specifies the data center zone where your cluster will be created.
      You can pick something from `this list
      <https://cloud.google.com/compute/docs/regions-zones/#available>`_
-     that is not too far away from your users.
+     that is not too far away from your users.                   
+   *  Note: A region is a geographical location used to host resources, comprising one or more zones.                                   
```

Comment:
@metonymic-smokey In rst markup format, a note is added using:

```
.. note::  This is a **note** box.
```

https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#colored-boxes-note-seealso-todo-and-warnings